src/smart-components/group/remove-group-modal.js

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import { withRouter } from 'react-router-dom';
+import { withRouter, useHistory } from 'react-router-dom';
```

Comment:
Same as with `add-group-wizard.js` this component no longer uses anything from decorator `withRouter`.